client/analytics/settings/setting.js

```diff
@@ -0,0 +1,135 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { Component } from '@wordpress/element';
+import PropTypes from 'prop-types';
+import { uniqueId } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import './setting.scss';
+
+class Setting extends Component {
+	renderInput = () => {
+		const { handleChange, name, inputType, options, value } = this.props;
+		const id = uniqueId( name );
+
+		switch ( inputType ) {
+			case 'checkboxGroup':
+				return options.map(
+					optionGroup =>
+						optionGroup.options.length > 0 && (
+							<div className="woocommerce-setting__options-group" key={ optionGroup.key }>
```

Comment:
Not 100% certain but thinking we might need `aria-labelledby` here [#](https://www.w3.org/TR/2016/WD-wai-aria-practices-1.1-20160317/examples/radio/radio.html)

Reply:
Added in https://github.com/woocommerce/wc-admin/pull/1364/commits/9a97ebfeab3474a6292b8c4fde12c359e4bd3b58 and tested with a screen reader.  I tied it to the setting label since the group label ("Custom Statuses" in the case of the 2nd group) may or may not be present here.